Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-slim-buster
+LABEL maintainer="Aspergillus Developers <aspergillus@mail.computational-biology.org>"
+
+WORKDIR /opt/nlisim
+COPY nlisim /opt/nlisim/nlisim
+COPY setup.py setup.cfg /opt/nlisim/
+COPY .git /opt/nlisim/.git
+RUN apt update \
+    && apt install -y git libgl1 \
+    && pip3 install . \
```

Comment:
This is going to leave some `__pycache__` and `.pyc` files as clutter in the built image. There are various techniques to getting rid of them.